exporter/otlphttpexporter/otlp.go

```diff
@@ -19,11 +19,13 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"go.opentelemetry.io/collector/client"
```

Comment:
Imports should be grouped together and sorted. This should go to the last group (internal import) IIRC. Run `make lint`, `make gofmt` and such - those should point this sort of errors